app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,84 @@
+    <string name="faq_how_it_works_text" tools:ignore="TypographyDashes"><![CDATA[<p>O %1$s converte links do Google Maps em links geo: que podem ser abertos por outros apps de mapa. Para criar um link geo:, coordenadas geográficas são necessárias. O %1$s as extrai da URL do Google Maps.</p><br><p>Porém, nem todas as URLs do Google Maps contêm coordenadas. Nestes casos, O %1$s te <b>pedirá permissão para se conectar ao Google</b> e obter as coordenadas dos cabeçalhos HTTP ou do documento HTML do link.</p><br><p>Mais precisamente, há três cenários como o %1$s converte a URL do Google Maps em uma URI geo: :</p><br><p><b>1.</b></p><br><p>Se a URL já contém coordenadas geográficas (por exemplo <tt>https://www.google.com/maps/place/Central+Park/data=!3d44.4490541!4d26.0888398</tt>), então elas são extraídas e nenhuma conexão é feita com os servidores do Google.</p><br><p><b>2.</b></p><br><p>Se a URL não contém coordenadas geográficas (por exemplo <tt>https://www.google.com/maps/place/Central+Park/</tt>), então o %1$s te perguntará se pode se conectar ao Google.</p><br><p>Se você permitir, então o %1$s irá fazer uma <b>solicitação HTTP de GET</b> ao Google Maps e extrai as coordenadas da resposta HTML. Você pode imaginá-lo como um <tt>curl https://www.google.com/maps/place/Central+Park/ | grep -E \'/@[0-9.,-]+\'</tt>.</p><br><p>Se você não permitir, então o %1$s criará um link geo: com o termo de pesquisa do lugar (por exemplo <tt>geo:0,0?q=Central%%20Park</tt>).</p><br><p><b>3.</b></p><br><p>Se a URL do Google Maps é um link curto (por exemplo <tt>https://maps.app.goo.gl/TmbeHMiLEfTBws9EA</tt>"), então o %1$s te perguntará se pode se conectar ao Google.</p><br><p>Se você permitir, então o %1$s irá fazer uma <b>solicitação HTTP de HEAD</b> ao link curto e lê o link completo dos cabeçalhos da resposta. Você pode imaginá-lo como um <tt>curl -I https://maps.app.goo.gl/TmbeHMiLEfTBws9EA | grep location:</tt>. E então o %1$s continua com o cenário 1 ou 2, dependendo se o link completo tem coordenadas geográficas ou não. No caso do cenário 2, outra conexão ao Google será feita, mas dessa vez sem perguntar.</p><br><p>Se você não permitir a conexão com o Google, então o %1$s cancela a criação do link geo:.</p><br><br><p>Para sempre permitir ou negar a conexão com o Google, em vez de sempre perguntar (padrão), visite as Preferências do app.</p>]]></string>
```

Comment:
I was gonna say this looks broken but it breaks the tokenazation in the original strings as well so /shrug